.env

```
@@ -64,7 +64,8 @@ LOCK_DSN=flock
 DIALOG_ORG_ID=e0d93630-acf7-4722-81e8-ff7d5fa64b66
 
 APP_STORAGE_SOURCE="local.storage"
+APP_MEDIAS_LOCATION="http://localhost:8000/storage"
```

Partout, utiliser media sans S ? (En anglais il n'y a jamais de S à media, ils considèrent que c'est déjà le pluriel latin de medium)

```suggestion
APP_MEDIA_LOCATION="http://localhost:8000/storage"
```